.storybook/stories/useAnimations.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react'
+import { Vector3 } from 'three'
+import { GLTF } from 'three/examples/jsm/loaders/GLTFLoader'
+import { withKnobs, select, number } from '@storybook/addon-knobs'
+
+import { Setup } from '../Setup'
+
+import { useAnimations, useGLTF, useMatcapTexture } from '../../src'
+
+export default {
+  title: 'Abstractions/useAnimations',
+  component: useAnimations,
+  decorators: [(storyFn) => <Setup cameraPosition={new Vector3(0, 0, 3)}>{storyFn()}</Setup>, withKnobs],
+}
+
+type GLTFResult = GLTF & {
+  nodes: {
+    YB_Body: THREE.SkinnedMesh
+    YB_Joints: THREE.SkinnedMesh
+    mixamorigHips: THREE.Bone
+  }
+  materials: {
+    YB_Body: THREE.MeshStandardMaterial
+    YB_Joints: THREE.MeshStandardMaterial
+  }
+}
+
+type ActionName = 'Dance' | 'Idle' | 'Strut'
+type GLTFActions = Record<ActionName, THREE.AnimationAction>
```

Comment:
This doesn't look like it's been used?

Reply:
Yeah you're right. I copied over a lot of the output from gltfjsx. I can remove these unused to clean up the story a bit.